text/0000-pack-buildpack-new-alternatives.md

```diff
@@ -0,0 +1,210 @@
+# Meta
+[meta]: #meta
+- Name: Pack Buildpack New to Support Alternatives
+- Start Date: 2022-03-10
+- Author(s): aidandelaney
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
```

Comment:
does this technically supersede https://github.com/buildpacks/rfcs/blob/main/text/0077-pack-buildpack-create.md#future-work?